cubids/cubids.py

```diff
@@ -401,28 +497,32 @@ def change_filename(self, filepath, entities):
         if len(filename) > 0:
             filename = sub_ses + "_" + filename + "_" + suffix + old_ext
         else:
+            # XXX: Why is filename here if it's an empty string?
             filename = sub_ses + filename + "_" + suffix + old_ext
 
         # CHECK TO SEE IF DATATYPE CHANGED
+        # XXX: Why/how would datatype change?
```

Comment:
datatypes can change during a `cubids-apply`. We ended up doing this at some point when we got data that had a mislabeled suffix

Reply:
Thanks! I'll update my comment.